app/components/footer.tsx

```diff
@@ -1,5 +1,7 @@
-import React from "react";
 import { Link } from "@remix-run/react";
+import styles from '../styles/index.css';
```

Comment:
When importing paths try to use absolute paths vs relative. In this case the import would be `import styles from '~/styles/index.css`